Assets/MRTK/Examples/Demos/HandTracking/Scenes/GridObjectCollectionFitterExample.unity

```diff
@@ -0,0 +1,4275 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
+OcclusionCullingSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_OcclusionBakeSettings:
+    smallestOccluder: 5
+    smallestHole: 0.25
+    backfaceThreshold: 100
+  m_SceneGUID: 00000000000000000000000000000000
```

Comment:
Can you move this script to MRTK/Examples/Experimental/GridObjectCollectionFitter/